tracetool/Modules/surface.py

```diff
@@ -0,0 +1,390 @@
+import os
```

Comment:
Why 'Modules' and 'Manifest' start from capital letter? Please, make it regular.